conda_build/build.py

```diff
@@ -288,6 +290,21 @@ def write_about_json(m, config):
             value = m.get_value('about/%s' % key)
             if value:
                 d[key] = value
+        # for sake of reproducibility, record some conda info
+        conda_info = json.loads(subprocess.check_output(['conda', 'info', '--json', '-s']))
+        d['conda_version'] = conda_info['conda_version']
+        d['conda_build_version'] = conda_info['conda_build_version']
+        d['conda_env_version'] = conda_info['conda_env_version']
+        d['offline'] = conda_info['offline']
+        d['channels'] = conda_info['channels']
```

Comment:
I don't think so.  There is an extra option for that:

```
--unsafe-channels  Display list of channels with tokens exposed.
```

This seems to have been added in 2015: https://github.com/conda/conda/blame/master/conda/cli/main_info.py#L74 so I think it's safe in this regard.